upup/models/cloudup/resources/addons/nodelocaldns.addons.k8s.io/k8s-1.12.yaml.template

```diff
@@ -99,9 +99,6 @@ data:
         bind {{ KubeDNS.NodeLocalDNS.LocalIP }}
         forward . __PILLAR__UPSTREAM__SERVERS__
         prometheus :9253
-        {{- if IsIPv6Only }}
```

Comment:
Should this be removed? Non-local lookups will be directly passed to AWS DNS, which will bypass dns64 if that is handled by coredns.

Reply:
As mentioned in the initial comment, the coredns in node-cache does not have the dns64 plugin. So leaving this in will cause the container to go into crashloop.